packages/yabridge-bin/yabridge-bin.pacscript

```diff
@@ -9,16 +9,16 @@ optdepends=("wine: 32-bit windows emulator"
 "wine64: 64-bit windows emulator")
 maintainer="rain1598 <guacamolek@yandex.com>"
 prepare() {
-          sudo install -D ./* /usr/lib
+          sudo install -D ./* /usr/lib/yabridge/
```

Comment:
```suggestion
          sudo install -D ./* "$STOWDIR/usr/lib/yabridge/"
```
It needs to be prefixed with `$STOWDIR`.

Basically my moving things under that directory, you are giving pacstall ability to manage it